fdbclient/CommitProxyInterface.h

```diff
@@ -514,11 +513,11 @@ struct ExclusionSafetyCheckRequest {
 };
 
 struct SSTeamChangeRequest {
-	constexpr static FileIdentifier file_identifier = 13852702; // TODO: how to choose this value?
-	// TODO: Add info about SS team changes
+	constexpr static FileIdentifier file_identifier = 13852702; // TODO: Need to choose an appropriate value here
```

Comment:
How to choose an appropriate value for FileIdentifier here?

Reply:
This is supposed to be a random number not used before. It seems `ExclusionSafetyCheckRequest` already uses this number, so you need to change to a different one.